<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Go</class>
 <widget class="QMainWindow" name="Go">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Go</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>100</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 100, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>600</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 100, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>390</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>70</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>801</width>
      <height>212</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton { 
font-size:50px;
color:yellow; 
border-style:outset;                 
border-width:6px; 
border-radius:80px;
background-color:rgba(0,100,255,200);
color:rgb(255, 255, 255);
font: 90 40pt &quot;m5x7&quot;; 
}

QPushButton:hover { 
font-size:50px;
color:green;
border-style:outset;                 
border-width:6px; 
border-radius:80px;
background-color:yellow; 
color:rgb(255, 255, 255);
font: 90 40pt &quot;m5x7&quot;;
}
QPushButton:pressed { 
color:orange;
border-width:3;
border-color:yellow;
border-style:solid;;
background-color:cyan; 
color:rgb(255, 255, 255);
font: 90 50pt &quot;m5x7&quot;;
}</string>
         </property>
         <property name="text">
          <string>网络</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton { 
font-size:50px;
color:yellow; 
border-style:outset;                 
border-width:6px; 
border-radius:80px;
background-color:rgba(0,100,255,200);
color:rgb(255, 255, 255);
font: 90 40pt &quot;m5x7&quot;; 
}

QPushButton:hover { 
font-size:50px;
color:green;
border-style:outset;                 
border-width:6px; 
border-radius:80px;
background-color:yellow; 
color:rgb(255, 255, 255);
font: 90 40pt &quot;m5x7&quot;;
}
QPushButton:pressed { 
color:orange;
border-width:3;
border-color:yellow;
border-style:solid;;
background-color:cyan; 
color:rgb(255, 255, 255);
font: 90 50pt &quot;m5x7&quot;;
}</string>
         </property>
         <property name="text">
          <string>单人</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>680</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
font-size:50px;
color:yellow; 
border-style:outset;                 
border-width:6px; 
border-radius:80px;
background-color:rgba(0,100,255,200);
color:rgb(255, 255, 255);
font: 90 40pt &quot;m5x7&quot;; 
}

QPushButton:hover { 
font-size:50px;
color:green;
border-style:outset;                 
border-width:6px; 
border-radius:80px;
background-color:yellow; 
color:rgb(255, 255, 255);
font: 90 40pt &quot;m5x7&quot;;
}
QPushButton:pressed { 
color:orange;
border-width:3;
border-color:yellow;
border-style:solid;;
background-color:cyan; 
color:rgb(255, 255, 255);
font: 90 50pt &quot;m5x7&quot;;
}</string>
    </property>
    <property name="text">
     <string>重开</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
